tcms/testruns/templates/testruns/get.html

```diff
@@ -56,6 +56,8 @@ <h2 class="card-pf-title" style="text-align: left">
         <h2 class="card-pf-title" style="text-align: left">
             <span class="fa fa-spinner"></span>{% trans 'Status' %}:
             <input class="bootstrap-switch" id="status_button" type="checkbox"
+                data-on-text="{% trans 'Running' %}"
+                data-off-text="{% trans 'Finished' %}"
```

Comment:
Revert this commit b/c it makes the UI inconsistent with other pages that use the ON/OFF button.